### PR TITLE
Fix/url html injection

### PR DESCRIPTION
**Pull request by JeremyDec**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-8092
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description

Block HTML tag potentially present in any url params (route path or url search query).

(See DTRSD-11881 internal ticket for more details)

### CODE REVIEW

**Review by antleblanc — COMMENTED** [review]
**question:** Can this logic be delegated to the `@ovh-ux/manager-core` module instead? (if make sense).

My main concern is to avoid code duplication.

Thanks!

File: packages/manager/modules/core/src/secure-url/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import angular from 'angular';
+import secureUrl from './secure-url.config';
+
+const moduleName = 'coreSecureUrl';
+
+angular.module(moduleName, []).run(secureUrl);
```
Comment (antleblanc) [inline packages/manager/modules/core/src/secure-url/index.js:6]: **question:** As `$transitions` provider is used, shouldn't `@uirouter/angularjs` module be injected as a module dependency?

File: packages/manager/modules/core/src/index.js [hunk 2]
```diff
@@ -28,6 +28,7 @@ import redirectionService from './redirection/redirection.service';
 import managerCoreUrlBuilder from './url-builder';
 
 import { registerConfigModule } from './config';
+import secureUrl from './secure-url/secure-url';
```
Comment (marie-j) [inline packages/manager/modules/core/src/index.js:31]: suggestion: 
```suggestion
import sanitizeUrl from './secure-url/secure-url';
```
is more explicit about what the module does

**Review by marie-j — DISMISSED** [review]

File: packages/manager/modules/core/src/index.js (hunk 2 above)
Comment (JeremyDec) [reply packages/manager/modules/core/src/index.js]: changed

**Review by antleblanc — DISMISSED** [review]

**Review by varun257 — DISMISSED** [review]

**Comment by JeremyDec** [comment]
rebase on `develop` since last approvals

**Review by marie-j — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**